pkg/rootless/rootless_linux.go

```diff
@@ -27,6 +27,8 @@ extern int reexec_userns_join(int userns);
 */
 import "C"
 
+const minimumMappings = 65536
```

Comment:
This seems a little drastic,  We have seen people attempting to sub-devide their allotment of UIDs.  So this change would only allow a user to use one container range in non root cases?  Have you actually seen containers fail with less then 65536 UIDs?

Reply:
the default value for `/proc/sys/kernel/overflowuid` is 65534 and from the strace here https://github.com/containers/libpod/issues/1092#issuecomment-410450029, it looks like `lchown` fails on using this value.

Reply:
So an image has 65535 inside of it?  Should minimumMappings then be set to /proc/sys/kernel/overflowuid  rather then hard coded?

Reply:
we could do that, but there is the risk that it will be too low (and I think this values is hard coded most of the time):

```
$ podman run --rm fedora grep  65534 /etc/passwd
nobody:x:65534:65534:Kernel Overflow User:/:/sbin/nologin
```

Reply:
anyway, we could probably read this value so to avoid another hard coded value.  I'll change the patch and submit a new version